modules/social_features/social_profile/social_profile.install

```diff
@@ -551,15 +551,44 @@ function social_profile_update_11002(): string {
   return $updateHelper->logger()->output();
 }
 
+/**
+ * Dummy update hook to prevent discrepancy.
+ */
+function social_profile_update_11003(array &$sandbox): void {
+  return;
+}
+
+/**
+ * Dummy update hook to prevent discrepancy.
+ */
+function social_profile_update_11004(array &$sandbox): void {
+  return;
+}
+
+/**
+ * Dummy update hook to prevent discrepancy.
+ */
+function social_profile_update_11005(array &$sandbox): void {
+  return;
+}
+
 /**
  * Install 'name' view mode.
  */
 function social_profile_update_11006(array &$sandbox): void {
   $module_path = drupal_get_path('module', 'social_profile');
+  $view_mode = EntityViewMode::load('profile.name');
+  if ($view_mode) {
```

Comment:
Please add some comments on why this is necessary.

Also why is this breaking? Do we know when this was added so now it's redundant here? I would suggest that we take some time to understand why this update is breaking.

Reply:
It will break if the view mode or display already exists. This could be the case when you come from 11.0.x (which had the update in the 11003 hook) and update to 11.1.x which will run update 11006 and results in a crash since the view mode already exists. By adding this check this won't break anymore. Hope this makes sense :)